cpp/daal/include/algorithms/pca/pca_explained_variance_batch.h

```diff
@@ -109,6 +109,9 @@ class Batch : public daal::algorithms::quality_metric::Batch
         input.set(eigenvalues, other.input.get(eigenvalues));
     }
 
+    // Add destructor
```

Comment:
May be it makes sense to remove this comment or replace it with
/** Default destructor*/ or smth like this